avocado/utils/distro.py

```diff
@@ -387,7 +387,7 @@ class DebianProbe(Probe):
 
     CHECK_FILE = "/etc/debian_version"
     CHECK_FILE_DISTRO_NAME = "debian"
-    CHECK_VERSION_REGEX = re.compile(r"(\d+)\.(\d+)")
```

Comment:
I've noticed that, for Debian stable, `/etc/debian_version` contains a numeric value, such as `12.7`.  On testing, because there is no numeric version number still determined, it contains `trixie/sid`.

I'm not sure it's a good idea to default to using code names instead of code names all over.  It may be a requirement for testing, but, numeric has advantages such as being able to do comparisons.  Or maybe I'm just not aware that Debian mostly disregards numeric versions.

My perception is that the best solution would be one where numeric values are given if available, and if not, code names as used as fallbacks.

Reply:
quick update that I have tried the above, and the unit tests below get in a stuck state, not able to further debug, not sure why, but seems that the solution of "ORing" in a regexp is not a good one.

```
unit-644-selftests/unit/utils/software_manager.py:Dpkg.test_is_valid: INTERRUPTED
unit-645-selftests/unit/utils/software_manager.py:Dpkg.test_is_not_valid: INTERRUPTED
```